### DISCUSSION TITLE
How to run the appium sample code in Android Emulator for Mac Big Sur (M1) chip?

### DISCUSSION THREAD

**Discussion by RekhaMahadevan** [appium/appium#15303] [Q&A]
Hi Team,

I have set up my appium server on my Mac Big Sur(M1) chip. I am able to run the sample code in the appium server on my terminal, however i cannot run the sample Hello World program in Android Emulator? If anyone can help me on this issue?

**Comment by KazuCocoa** [comment]
I don't know if Android Studio already supports M1 mac. You should make sure it in Google's developer pages
e.g. https://androidstudio.googleblog.com/2020/12/android-emulator-apple-silicon-preview.html

**Reply by RekhaMahadevan** [reply]
Hi @KazuCocoa Thanks for your update, I have already tried the M1 preview version of Android Emulator. However, I am unable to open the chrome browser on the Emulator. I can see only a web view tester as shown in the below: which is not properly working for browser pages:
<img width="446" alt="Screenshot 2021-04-28 at 13 35 14" src="https://user-images.githubusercontent.com/81163196/116404492-93fd3280-a826-11eb-866e-29e3fccc2c35.png">

**Comment by KazuCocoa** [comment]
>  I am unable to open the chrome browser on the Emulator

Did the emulator have google api version? If no, probably Google hasn't provided a version which has Chrome/Google API stuff.

**Reply by RekhaMahadevan** [reply]
Am not sure how to check the google API version for emulator?Do you know?

**Reply by KazuCocoa** [reply]
Usually you can choose images in avdmanager
